File: src/main/java/com/examples/addone/incorrect/java/deadcode/Question.java
package com.examples.addone.incorrect.java.deadcode;

import edu.illinois.cs.cs125.questioner.lib.Incorrect;

@Incorrect(reason = "deadcode")
public class Question {
  int addOne(int value) {
    if (value == 1234567) {
      return 1234568;
    }
    return value + 1;
  }
}

File: src/main/java/com/examples/withfeaturecheck/incorrect/kotlin/usesloop/Question.kt
package com.examples.withfeaturecheck.incorrect.kotlin.usesloop

import edu.illinois.cs.cs125.questioner.lib.Incorrect

@Incorrect(reason = "features")
class Question {
  fun sumTo(range: Int): Int {
    var sum = 0
    (0..range).forEach { i ->
      sum += i
    }
    return sum
  }
}